vault/core.go

```diff
@@ -686,9 +673,6 @@ func (c *Core) SecretProgress() (int, string) {
 func (c *Core) ResetUnsealProcess() {
 	c.stateLock.Lock()
 	defer c.stateLock.Unlock()
-	if !c.sealed {
```

Comment:
Note -- I came across this while looking for c.sealed, but I think it's an error anyways. I can't see a reason why we shouldn't reset this when asked, regardless of seal state.